test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -1062,22 +1062,32 @@ def test_qlinear_int8_mixed_bf16(self):
         """
         self._qlinear_cpu_test_helper(int8_mixed_bf16=True)
 
-    def _qlinear_unary_cpu_test_helper(self, int8_mixed_bf16=False):
+    def _qlinear_unary_cpu_test_helper(
+        self, unary_op=torch.nn.ReLU(), int8_mixed_bf16=False
+    ):
         class M(torch.nn.Module):
-            def __init__(self, use_bias):
+            def __init__(self, use_bias, post_op_algo):
                 super().__init__()
                 self.linear = torch.nn.Linear(4, 4, use_bias)
-                self.unary_fn = torch.nn.ReLU()
+                if unary_op == torch.nn.GELU():
+                    self.unary_fn = copy.deepcopy(unary_op(approximate=post_op_algo))
+                else:
+                    self.unary_fn = copy.deepcopy(unary_op)
                 self.linear2 = torch.nn.Linear(4, 4, use_bias)
-                self.unary_fn2 = torch.nn.ReLU()
+                if unary_op == torch.nn.GELU():
```

Comment:
2 different GELU instances should not be same always.

Reply:
Thanks for the comment and changed accordingly.